src/export/meetbouten/__init__.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from export.api import API
+from export.meetbouten.config import MeetboutExportConfig, MetingenExportConfig, ReferentiepuntenExportConfig
+from export.meetbouten.types import type_convert
+
+
+CONFIG_MAPPING = {
+    'meetbouten': MeetboutExportConfig,
+    'metingen': MetingenExportConfig,
+    'referentiepunten': ReferentiepuntenExportConfig,
+}
+
+
+def _export_entity(entity, format):
+    """Exports a single entity
+
+    The export file specifications can be found in:
+        https://www.amsterdam.nl/stelselpedia/meetbouten-index/producten/prodspecs-mebo/
+
+    Headers:       None
+    Separator:     |
+    String marker: $$
+
```

Comment:
This info is no longer appropriate for this method

Reply:
Discussed to take into account on next catalog export